apidoc/api/v1/subscription_line_items.md

```diff
@@ -2,7 +2,7 @@
 SolidusSubscriptions::LineItems are analogous to Spree::LineItem. They indicate
 the details of a subscription prior to a purchase being made.
 
-## PATCH `/api/v1/line_item/:id`
+## PATCH `subscriptions/api/v1/line_item/:id`
```

Comment:
Could we add a slash here and to the other headers?

Reply:
Fixed!